app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffectHandlerTest.kt

```diff
@@ -95,4 +96,18 @@ class BloodSugarEntryEffectHandlerTest {
     verify(ui).showBloodSugarLowError()
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `show blood sugar entry screen when sheet back button is clicked and date entry screen is active`() {
```

Comment:
Can we reword this test to match the rest?

Ex: `show blood sugar entry screen when show blood sugar entry screen effect is received`.

I also do not feel it is necessary to add the `and date entry screen is active` part because this is a UI detail and there is no reference to this in the test body itself.